### DISCUSSION TITLE
module missing nuget/private nuget feed dependency

### DISCUSSION THREAD

**Discussion by ipavlu** [cake-build/cake#3030] [General]
Hi,

I created a module based on cake module construction video tutorial.
It worked when it was simple.
But when I tied some functionality to nuget or more specifically private nuget feed,
then I found myself in trouble. The module seems to be missing nuget packaged dependencies.

I installed the module by -bootstrap, I have set --nuget_loaddependencies=true and also in config, nothing helped...

config.cake and nuget.config contain all necessary parts.
The module was installed by --bootstrap from private nuget feed configured in nuget.config, so it works.
It just installs the nugget package of the module, the dll, but dependencies that are in sam feed are not present.

**Comment by augustoproiete** [comment]
@ipavlu Are you able to provide a link to a repro example?
